qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/modules/kubernetes/services/clearml/chart/values.yaml

```diff
@@ -226,6 +226,8 @@ redis: # configuration from https://github.com/bitnami/charts/blob/master/bitnam
   master:
     name: "{{ .Release.Name }}-redis-master"
     port: 6379
+    nodeSelector:
```

Comment:
How is the node being labeled "app: clearml"?

Reply:
Or this an assumption that the node will have this label?

Reply:
> How is the node being labeled "app: clearml"

good point. It's a default value set in the variable.tf file that propagest to the chart's values.yaml (see [here](https://github.com/Quansight/qhub/blob/29f398af3542143751d52252b8a2204cd9c1e5cf/qhub/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/infrastructure/modules/kubernetes/services/clearml/main.tf#L8))). 


> Or this an assumption that the node will have this label?

It is a requirement. We manually set this label for our deployment. We are not sure if this is automated when QHub deploys clearml when using the cloud providers (as described [here](https://docs.qhub.dev/en/stable/source/admin_guide/clearml.html?highlight=clearml#create-a-node-group)).

Reply:
that drew our attention to missing redis/mongdb dynamic value setting. fixed now.